src/Confluent.Kafka/AdminClient.cs

```diff
@@ -123,6 +123,20 @@ private List<DescribeConfigsReport> extractResultConfigs(IntPtr configResourcesP
             return result;
         }
 
+        private static List<DeleteGroupReport> extractDeleteGroupReport(IntPtr eventPtr)
```

Comment:
`extraceDeleteConsumerGroupsReport`

groups is plural and use the word consumer.

Reply:
Change to plural but avoid adding consumer - see comment above.